mllib/src/test/java/org/apache/spark/mllib/clustering/JavaLDASuite.java

```diff
@@ -110,8 +112,8 @@ public void distributedLDAModel() {
     assertEquals(roundedLocalTopicSummary.length, k);
 
     // Check: log probabilities
-    assert(model.logLikelihood() < 0.0);
-    assert(model.logPrior() < 0.0);
```

Comment:
Java's `assert` should not used in unit tests, which could be turned off via JVM flags.